CMakeLists.txt

```diff
@@ -13,6 +13,10 @@
 
 PROJECT(libgit2 C)
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
I don't think we want to include this in our CMakeLists.txt, as it would require everybody to build libgit2 with Conan.

Reply:
I agree. I plan to add this via an option. This is still in MVP phase.

Reply:
Tip (for further recipes): Use `tools.replace_in_file()` within the Conan recipe or use an optional Conan CMake wrapper so that Conan stays optional